libsolidity/ast/AST.h

```diff
@@ -814,6 +822,7 @@ class VariableDeclaration: public Declaration
 	bool m_isStateVariable; ///< Whether or not this is a contract state variable
 	bool m_isIndexed; ///< Whether this is an indexed variable (used by events).
 	bool m_isConstant; ///< Whether the variable is a compile-time constant.
+	bool m_isVirtual; ///< Whether the variable is virtual and can be overridden
```

Comment:
Haha, I *would* suggest to (redundantly) member-initialize this to ``false`` as well, but looking at the lines above this would be somewhat superficial :-D. Can you just member-initialize all of these ``bool``s to ``false`` anyways and also ``m_location`` below to ``Location::Unspecified``? We have to start doing it consistently some time, so why not now...

Reply:
I haven't done this consistently myself in the recent past, though, I must admit, since I just read the not-member-initialized ``FunctionDefinition::kind`` above :-). But still.